framework/sessions/__init__.py

```diff
@@ -160,7 +160,7 @@ def before_request():
         if not util_time.throttle_period_expired(user_session.date_created, settings.OSF_SESSION_TIMEOUT):
             if user_session.data.get('auth_user_id') and 'api' not in request.url:
                 OSFUser = apps.get_model('osf.OSFUser')
-                OSFUser.objects.filter(guids___id=user_session.data['auth_user_id']).update(date_last_login=timezone.now())
+                OSFUser.objects.filter(guids___id=user_session.data['auth_user_id']).invalidated_update(date_last_login=timezone.now())
```

Comment:
Doesn't need to happen in this PR, but it might be worth overriding `update` on `BaseModel's` QuerySet so that is always calls `invalidated_update`.